docs/reference/config.md

```diff
@@ -517,9 +517,26 @@ nav_order: 3
 |maxAge|The maximum time to retain old log files based on the timestamp encoded in their filename.|[`time.Duration`](https://pkg.go.dev/time#Duration)|`<nil>`
 |maxBackups|Maximum number of old log files to retain|`int`|`<nil>`
 |noColor|Force color to be disabled, event when TTY output is detected|`boolean`|`<nil>`
+|reportCaller|Enables the report caller for including the calling file and line number, as well as the calling function. Only works for JSON formatted logs.|`boolean`|`<nil>`
```

Comment:
I wonder if `includeCodeInfo` would be clearer here, or similar?

Reply:
Fine with `includeCodeInfo` does sound a bit clearer than `reportCaller` 👍🏻